src/main/java/org/jboss/arquillian/extension/jacoco/client/ApplicationArchiveInstrumenter.java

```diff
@@ -47,6 +47,10 @@
    @Inject
    private Instance<JacocoConfiguration> config;
 
+   public void setConfig(Instance<JacocoConfiguration> config) {
+      this.config = config;
+   }
```

Comment:
We could I believe, but this would make it even more hacky (that's how `@Inject` works underneath anyway). Maybe just comment with sth like
```
// FIXME used for testing only. remove once constructor injection supported
```